lti_provider/migrations/0005_auto_20171009_1234.py

```diff
@@ -20,9 +40,14 @@ class Migration(migrations.Migration):
             model_name='lticoursecontext',
             name='uuid',
         ),
-        migrations.AddField(
+        PostgresAddLMSCourseContext(
             model_name='lticoursecontext',
             name='lms_course_context',
-            field=models.TextField(null=True, unique=True),
+            field=models.TextField(null=True, unique=True)
```

Comment:
I have some questions about what's going on here. Are you intending to keep the `lms_course_context`field as a TextField on Postgres to minimize the potential for issues here? I find that a little confusing, since this field is still specified as a CharField in models.py.

Reply:
The 0006 migration updates the field to CharField.

Reply:
Ah okay, I understand now why this is necessary.

Reply:
I probably should elaborate: I went this route to maintain the integrity of the postgres migration path. Though slim, mucking with migrations can cause some side-effects that I felt this approach avoids.